model/event/admin_event_test.go

```diff
@@ -28,7 +28,10 @@ func TestAdminEventSuite(t *testing.T) {
 }
 
 func (s *AdminEventSuite) SetupTest() {
-	db.Clear(AllLogCollection)
+	err := db.Clear(AllLogCollection)
+	if err != nil {
+		panic(err)
```

Comment:
I would just s.Require().NoError() 

which has the same effect and more clear error messages.

Reply:
Done.